package.json

```diff
@@ -37,7 +37,6 @@
     "ember-cli-is-package-missing": "^1.0.0",
     "ember-cookies": "^0.1.0",
     "ember-fetch": "^2.1.0 || ^3.0.0",
-    "ember-getowner-polyfill": "^1.1.0 || ^2.0.0",
```

Comment:
This is actually needed to be able to support old Ember versions (ESA still supports Ember down to 1.12). I'm not sure why the tests don't fail without this…